core/src/io.jl

```diff
@@ -165,3 +166,18 @@ function write_flow_output(model::Model)
     mkpath(dirname(path))
     Arrow.write(path, table; compress = :lz4)
 end
+
+function write_control_output(model::Model)
+    config = model.config
+    record = model.integrator.p.control.record
+
+    if length(record.time) > 0
```

Comment:
I'm not sure what the behaviour of the other arrow writing code is. Do we write empty tables when asked? We should do the same here.

Reply:
I think basin and flow tables are always written, but if they are empty the model is rather trivial.